<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_stats</class>
 <widget class="QDialog" name="Dialog_stats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>103</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>73</y>
     <width>261</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>1) Chromosomes Fitness Variation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>2) N-gram Statistics</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>103</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>View Plot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>103</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>View Plot</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_stats</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
